pkgs/development/python-modules/pyxl3/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, unittest2
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "pyxl3";
```

Comment:
This probably should be disabled for python 2.7?

Reply:
Fixed.